selfdrive/car/car_helpers.py

```diff
@@ -180,6 +182,9 @@ def fingerprint(logcan, sendcan, num_pandas):
 
   cloudlog.event("fingerprinted", car_fingerprint=car_fingerprint, source=source, fuzzy=not exact_match, cached=cached,
                  fw_count=len(car_fw), ecu_responses=list(ecu_rx_addrs), vin_rx_addr=vin_rx_addr, error=True)
+
+  global_ti.saved_candidate = car_fingerprint
```

Comment:
is their is reason why we need this? why cannot we not use dp-car selector ?

Reply:
Ah right, sorry, I'm not all that familiar with the way dragonpilot works. I think you're right, and it looks like we can also get rid of global_ti.py